compiler/src/dotty/tools/dotc/transform/MoveStatics.scala

```diff
@@ -19,6 +20,8 @@ class MoveStatics extends MiniPhaseTransform with SymTransformer { thisTransform
   import tpd._
   override def phaseName = "moveStatic"
 
+  // This phase moves methods around(in infortransform) so it may need to make other methods public
+  override def runsAfter: Set[Class[_ <: Phase]] = Set(classOf[ExpandPrivate])
```

Comment:
You moved `MoveStatic` _before_ `ExpandPrivate` but you added a requirement that it should be run `after`, so every test fails.